packages/ez-conf-lib/ez-conf-lib.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Simple generator of conf-<pkg>.config files"
+description:
+    "This package provides a simplistic utility that \
+     helps writing virtual packages used to detect system libraries"
+authors: ["Nicolas Berthier"]
+maintainer: "Nicolas Berthier <m@nberth.space>"
+homepage: "https://github.com/nberth/ez-conf-lib"
+dev-repo: "git+https://github.com/nberth/ez-conf-lib.git"
+bug-reports: "https://github.com/nberth/ez-conf-lib/issues"
+license: "GPL-3.0-only"
+build: [
+  [ "sh" "-ecx" "sed -e 's_@@ez-conf-lib:lib@@_%{_:lib}%_g' \
+                     ez-conf-lib.config.in > ez-conf-lib.config" ]
```

Comment:
This is utterly broken for local opam switches in `_opam` directories: https://github.com/nberth/ez-conf-lib/issues/3.

Reply:
Thanks for spotting that beginner mistake (arf… that's what happens when you hack things too quickly). #25313 should bring a version that does not rely on `sed`.